lib/auth/auth.go

```diff
@@ -3112,9 +3112,6 @@ func (a *Server) CreateRegisterChallenge(ctx context.Context, req *proto.CreateR
 		if err != nil {
 			return nil, trace.Wrap(err)
 		}
-
-	default:
-		return nil, trace.BadParameter("either a token or an MFA response are required")
```

Comment:
Is this change necessary? Just curious.

Reply:
Only to make this change work - https://github.com/gravitational/teleport/pull/38451#discussion_r1499391831

Should I revert?

Reply:
Nope, it's fine.